inc/functions/options.php

```diff
@@ -413,6 +413,7 @@ function get_rocket_exclude_defer_js() {
 		'google.com/recaptcha',
 		'widget.reviews.co.uk',
 		'lib/admin/assets/lib/webfont/webfont.min.js'
```

Comment:
Forgot to add a comma in here:
`'lib/admin/assets/lib/webfont/webfont.min.js',`

Reply:
Oops! 😅 

Done!